package.json

```diff
@@ -80,7 +80,6 @@
     "jest-extended": "^1.2.0",
     "jest-mock-extended": "^2.0.4",
     "lint-staged": "11.1.2",
-    "nx": "12.9.0",
```

Comment:
Why `nx` is removed from the `devDependencies`? 

I think we better have it in as many scripts depend on it's binaries.

Please roll it back